src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/SystemRemindApplicant1ApplicationApprovedTask.java

```diff
@@ -58,21 +58,13 @@ public void run() {
             for (final CaseDetails caseDetails : casesInAwaitingApplicant1Response) {
                 try {
                     final CaseData caseData = objectMapper.convertValue(caseDetails.getData(), CaseData.class);
-                    LocalDate dueDate = caseData.getDueDate();
-                    log.info("Due Date for case id: {} is {}", caseDetails.getId(), caseData.getDueDate());
-
-                    if (dueDate == null) {
-                        log.error("Ignoring case id {} with created on {} and modified on {}, as due date is null",
-                            caseDetails.getId(),
-                            caseDetails.getCreatedDate(),
-                            caseDetails.getLastModified()
-                        );
-                    } else {
-                        if (!dueDate.isAfter(LocalDate.now())
-                            && !caseData.getApplication().isApplicant1ReminderSent()
-                        ) {
-                            notifyApplicant1(caseDetails, caseData, dueDate, user, serviceAuthorization);
-                        }
+                    LocalDate reminderDate = caseData.getDueDate().minusWeeks(1);
+                    log.info("Reminder Date for case id: {} is {}", caseDetails.getId(), caseData.getDueDate());
+
+                    if (!reminderDate.isAfter(LocalDate.now())
+                        && !caseData.getApplication().isApplicant1ReminderSent()
```

Comment:
could you please add a log stating case id was not eligible for remind app1 application hence skipping it helps when debugging to see if the case was picked or not